src/sonic-yang-models/tests/yang_model_tests/tests_config/kdump.json

```diff
@@ -5,7 +5,10 @@
                 "config": {
                     "enabled": "true",
                     "num_dumps": "3",
-                    "memory": "0M-2G:256M,2G-4G:256M,4G-8G:384M,8G-:448M"
+                    "memory": "0M-2G:256M,2G-4G:256M,4G-8G:384M,8G-:448M",
+                    "remote": "true",
+                    "ssh_string" : "username@ipadress",
```

Comment:
This can't be the sample config, please use the actual configuration e.d user1@10.0.0.2 and actual private key path, this is the problem of using the string type.

Reply:
Noted, Updating as per suggestion